extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegesBuilder.scala

```diff
@@ -411,6 +469,26 @@ object PrivilegesBuilder {
         outputObjs += tablePrivileges(table, actionType = actionType)
         buildQuery(getQuery, inputObjs)
 
+      case "AppendData" => // DsV2 insert
```

Comment:
the comment here is useless.

How about we group v2 commands together with or without a new method

Reply:
I prefer group v2commands in set or enum , but not to introduce new method for handling them. Stay in one method is good enough for now.